File: app/src/main/java/org/michaelbel/moviemade/rest/api/DISCOVER.java
package org.michaelbel.moviemade.rest.api;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface DISCOVER {

    @GET("discover/movie?")
    Observable<?> movieDiscover(

    );

    @GET("discover/tv?")
    Observable<?> tvDiscover(

    );
}

File: app/src/main/java/org/michaelbel/moviemade/data/service/DISCOVER.kt
package org.michaelbel.moviemade.data.service

import io.reactivex.Observable
import retrofit2.http.GET

interface DISCOVER {

    @GET("discover/movie?")
    fun movieDiscover(): Observable<*>

    @GET("discover/tv?")
    fun tvDiscover(): Observable<*>
}